pkg/aws/aws_suite_test.go

```diff
@@ -53,14 +55,11 @@ func newMachine(
 		},
 	}
 
-	// Don't initialize providerID and node if setMachineIndex == -1
+	// Don't initialize providerID if setMachineIndex == -1
 	if setMachineIndex != -1 {
 		machine.Spec = v1alpha1.MachineSpec{
 			ProviderID: fmt.Sprintf("aws:///eu-west-1/i-0123456789-%d", setMachineIndex),
 		}
-		machine.Status = v1alpha1.MachineStatus{
-			Node: fmt.Sprintf("ip-%d", setMachineIndex),
-		}
```

Comment:
we should replace this with `setting the node label`

Reply:
done